packages/webpack-cli/lib/groups/StatsGroup.js

```diff
@@ -5,7 +5,9 @@ const logger = require('../utils/logger');
  */
 class StatsGroup extends GroupHelper {
     static validOptions() {
-        return ['none', 'errors-only', 'minimal', 'normal', 'detailed', 'verbose', 'errors-warnings'];
+        let validArrayString = ['none', 'errors-only', 'minimal', 'normal', 'detailed', 'verbose', 'errors-warnings'];
+        let validArrayObject = [{ verbose: true }, { verbose: true, stats: 'verbose' }];
```

Comment:
Why we need it here? We don't support objects in webpack-cli

Reply:
I used that for tests. It deals with when a user provide `--stats verbose --verbose` both flags together.

Reply:
Put it in tests - you can sue mocking